routers/routes/routes.go

```diff
@@ -479,7 +479,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 		// FIXME: should use different URLs but mostly same logic for comments of issue and pull reuqest.
 		// So they can apply their own enable/disable logic on routers.
 		m.Group("/issues", func() {
-			m.Combo("/new", repo.MustEnableIssues).Get(context.RepoRef(), repo.NewIssue).
```

Comment:
Seems no replacement for this removal?

Reply:
Handler in parent func, line 509, use `context.CheckUnit(models.UnitTypeIssues)`